src/core/pre.js

```diff
@@ -59,3 +59,5 @@ const errNoRet = () => {
     "Assertion error: control reached end of function without return",
   );
 };
+
+Module.reportUndefinedSymbols = () => {};
```

Comment:
Why is this required?

Reply:
In a nodylink build, `Module.reportUndefinedSymbols` is not a thing, so we make it a no op.